editor/src/components/canvas/canvas-strategies/strategies/flex-reparent-to-flow-strategy.spec.browser2.tsx

```diff
@@ -0,0 +1,637 @@
+    const targetFlexParentEnd = {
+      x: targetFlexParentRect.x + targetFlexParentRect.width - 15,
+      y: targetFlexParentRect.y + targetFlexParentRect.height / 2,
+    }
+    const flexChildToReparent = await renderResult.renderedDOM.findByTestId('flexchild3')
+    const flexChildToReparentRect = flexChildToReparent.getBoundingClientRect()
+    const flexChildToReparentCenter = {
+      x: flexChildToReparentRect.x + flexChildToReparentRect.width / 2,
+      y: flexChildToReparentRect.y + flexChildToReparentRect.height / 2,
+    }
+
+    await renderResult.getDispatchFollowUpActionsFinished()
+    const dragDelta = windowPoint({
+      x: targetFlexParentEnd.x - flexChildToReparentCenter.x + 5,
+      y: targetFlexParentEnd.y - flexChildToReparentCenter.y,
+    })
+
+    dragElement(renderResult, 'flexchild3', dragDelta, cmdModifier)
+
+    await renderResult.getDispatchFollowUpActionsFinished()
+
+    expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
+      makeTestProjectCodeWithSnippet(`
+    <div
+      style={{
+        position: 'absolute',
+        width: 700,
+        height: 600,
+      }}
+      data-uid='container'
+      data-testid='container'
+    >
+      <div
+        style={{
+          position: 'absolute',
+          width: 250,
+          height: 500,
+          left: 0,
+          top: 0,
+          backgroundColor: 'blue',
+        }}
+        data-uid='flowparent1'
+        data-testid='flowparent1'
+      >
+        <div
+          style={{
+            width: 100,
+            height: 100,
+            backgroundColor: 'purple',
+            display: 'inline-block',
+          }}
+          data-uid='flowchild1'
+          data-testid='flowchild1'
+        />
+        <div
+          style={{
+            width: 100,
+            height: 100,
+            backgroundColor: 'pink',
+            display: 'inline-block',
+          }}
+          data-uid='flowchild2'
+          data-testid='flowchild2'
+        />
+        <div
+          style={{
+            width: 100,
+            height: 100,
+            backgroundColor: 'teal',
+            display: 'inline-block',
+          }}
+          data-uid='flexchild3'
+          data-testid='flexchild3'
+        />
+      </div>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          position: 'absolute',
+          width: 250,
+          height: 500,
+          left: 350,
+          top: 0,
+          backgroundColor: 'lightgreen',
+        }}
+        data-uid='flexparent2'
+        data-testid='flexparent2'
+      >
+        <div
+          style={{
+            width: 100,
+            height: 100,
+            backgroundColor: 'red',
+          }}
+          data-uid='flexchild4'
+          data-testid='flexchild4'
+        />
+      </div>
+    </div>
+  `),
+    )
+  })
+
+  // TODO: enable when insertion to zero position is fixed
```

Comment:
what is this TODO about?

Reply:
Sorry, outdated comment, the test is already enabled under it